apps/wing/src/commands/lsp.ts

```diff
@@ -136,9 +136,14 @@ export async function lsp() {
     for (const rd of raw_diagnostics) {
       if (rd.span) {
         const diagnosticUri = "file://" + rd.span.file_id;
+        let message = rd.message;
```

Comment:
Test for this maybe?

Reply:
Unfortunately we don't have any fixture/system for e2e testing of the lsp and unit testing this particular function wouldn't be too useful imo